fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
I would like to register my complaint that building with an eye towards hypothetical future requirements at the expense of an elegant implementation against our current implementation often costs more time than it saves.